contents/stats.md

```diff
@@ -40,7 +40,8 @@ And by using the observed data we can use inference to gain knowledge about the
 fig = statistics_graph()
 caption = "Statistics vs Probability"
 label = "stats_vs_prob"
-Options(fig; filename=label, caption, label)
+link_attributes = "width=60%" # hide
```

Comment:
```suggestion
link_attributes = "width=60%"
```
It is not necessary here, right?

Reply:
Yeah, it will not output anything since it isn't inside a string and we are not calling `sco(s)`.